data.js

```diff
@@ -0,0 +1,31 @@
+export const data = {
+  "header": {
+    "title-team": "The creative crew",
+    "description-team-title": "Who we are",
+    "description-team-text": "We are team of creatively diverse. driven. innovative individuals working in various locations from the world."
+  },
+  "team-member-section": {
+    "team-member-card": [
+      {
+        "image": "/home/susanar/Documentos/Programacion_2021_1/Layouts/my-team-page/photo4.png",
+        "name-title": "Skylah Lu",
+        "charge-text": "UX DESIGNER"
+      }
+    ]
+  },
+  "footer": {
+    "social-media":
+      [
+        {
+          "href": "https://instagram.com/ve5pula",
+          "image": "https://www.flaticon.es/svg/static/icons/svg/725/725339.svg",
+          "alt": "Instagram Image"
+        },
+        {
+          "href": "https://twitter.com/ve5pula",
+          "image": "https://www.flaticon.es/svg/static/icons/svg/725/725372.svg",
+          "alt": "Twitter Image"
+        }
+      ]
+  }
+}
```

Comment:
I think you need to repeat this data as many times as cards are on the page or create different cards to populate the team page properly.

Reply:
10e33b8